### ISSUE TITLE
About train details

### ISSUE THREAD

**Issue by sfwyly** [furrypython/PConv-TensorFlow2#1]
Hi, your result is surprising, but the results I ran were very blurry and artificial trace in the holes region, why?
And I think you use (batch: image + mask )as input ,but i use( batch image + 1 mask) as inputs, Is it possible for this to lead to my unsatisfactory result? However, the problem still exists after I used (batch : image + mask). In addition, in accordance with results of the paper, I turned up some of the style loss is still futile. Thanks!
